third_party/amd/lib/TritonAMDGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -361,6 +361,21 @@ void createBarrier(ConversionPatternRewriter &rewriter, Location loc,
 }
 } // namespace
 
+static LLVM::AtomicOrdering llvmMemoryOrdering(MemSemantic memOrdering) {
```

Comment:
Nit: normally we name functions with a verb prefix. Can we call it `getMemoryOrdering` here?

Reply:
Thanks. changed accordingly.

Reply:
fixed. Thanks